CONTRIBUTING.md

```diff
@@ -8,8 +8,9 @@ The following is a set of guidelines for contributing to GMLC-TDC and associated
 
 ## Licensing
 
-The Utilities library is distributed under the terms of the BSD-3 clause license. All new
-contributions must be made under this [LICENSE](LICENSE) in accordance with the Github [terms of service](https://help.github.com/en/articles/github-terms-of-service#6-contributions-under-repository-license), which uses inbound=outbound policy. By submitting a pull request you are acknowledging that you have the right to license your code under these terms.
+The networking library is distributed under the terms of the BSD-3 clause license.
+All new contributions must be made under this [LICENSE](LICENSE) in accordance with the github [terms of service](https://help.github.com/en/articles/github-terms-of-service#6-contributions-under-repository-license), which uses inbound=outbound policy.
```

Comment:
Maybe `GitHub`? At least, that's how GitHub capitalizes their own name.

```suggestion
All new contributions must be made under this [LICENSE](LICENSE) in accordance with the GitHub [terms of service](https://help.github.com/en/articles/github-terms-of-service#6-contributions-under-repository-license), which uses inbound=outbound policy.
```

Reply:
I will try that in the next update see if that works